docs/Open Data Model/Open Data Model.md

```diff
@@ -0,0 +1,755 @@
+
+ **- Reduction of mean time to incident detection & resolution (MTTR)**
+
+One of the challenges organizations face today is detecting threats early enough to minimize adverse impacts. This stems from the limitations previously discussed with regards to limited analytics. It can also be attributed to the fact that most of the investigative queries often take hours or days to return results. Legacy technologies can’t offer or have a central data store for facilitating such investigations due to their inability to store and serve the massive amounts of data involved. This cripples incident investigations and results in MTTRs of many weeks or months, meanwhile the adverse impacts of the breach are magnified, thus making the threat harder to eradicate.
+
+Apache Spot (incubating) addresses these gaps by providing the capability for a central data store that houses ALL the data needed to facilitate an investigation, returning investigative query results in seconds and minutes (vs. hours and days). Spot can effectively reduce incident MTTR and reduce adverse impacts of a breach.
+
+ **- Threat Hunting**
+
+It’s become necessary for organizations to “hunt” for active threats as traditional passive threat detection approaches are not sufficient. “Hunting” involves performing ad-hoc searches and queries over vast amounts of data representing many weeks and months’ worth of events, as well as applying ad-hoc / tune algorithms to detect the needle in the haystack. Traditional systems do not perform well for these types of activities as the query results sometimes take hours and days to be retrieved. These traditional systems also lack the analytic flexibility to construct the necessary algorithms and logic needed.
+
+Apache Spot (incubating) addresses these gaps in the same ways it addresses others; by providing a central data store with the needed analytic frameworks that scale to the needed workloads.
+
+**Data Model**
+----------
+In order to provide a framework for effectively analyzing data for cyber threats, it is necessary to collect and
+analyze standard security event logs/alerts and contextual data regarding the entities referenced in these logs/alerts. The most common entities include network, user and endpoint, but there are others such as file.
+
+In the diagram below, the raw event tells us that user “jsmith” successfully logged in to an Oracle database from the IP address 10:1.1.3. Based on the raw event only, we don’t know if this event is a legitimate threat or not. After injecting user and endpoint context, the enriched event tells us this event is a potential threat that requires further investigation.
+
+![Screen Shot 2016-09-22 at 1.11.28 PM.png](CybersecurityOpenDataModel0%204-3_files/image001.jpg)
```

Comment:
Can we write out this text as well please? I do understand that the original doc had a screenshot but those are really hard to manage to it'd be good to textify that.